src/Connection.ts

```diff
@@ -1680,11 +1680,11 @@ export class Connection<
 
 	/**
 	 * Read the directory of an adapter.
-	 * @param adapterName The adapter name.
+	 * @param {string} namespace (this may be the adapter name, the instance name or the name of a storage object within the adapter).
```

Comment:
```suggestion
	 * @param namespace (this may be the adapter name, the instance name or the name of a storage object within the adapter).
```
Adding types to JSDoc is unnecessary when using TS.